models/sessionization/segment_web_sessions__initial.sql

```diff
@@ -120,6 +120,7 @@ tiers as (
 mapped as (
 
     select
+        distinct
```

Comment:
Can you talk me through the `distinct` here — is this solving a separate issue (if so, it should be in a separate PR!). `distinct`s are expensive, so I'd prefer to avoid them if possible :)

Reply:
Yeah the distinct here is to prevent the duplicate pintrest listed above. I think it is bold to assume that the mappings from referrer_mapping.csv will be unique. 

```select
        distinct
        tiers.*,
        referrer_mapping.medium as referrer_medium,
        referrer_mapping.source as referrer_source

    from tiers

    left join referrer_mapping on tiers.referrer_host = referrer_mapping.host
```

The join on host will duplicate the tiers and will cause non-unique referrals when they were unique prior to the join